build-setup-nolog.sh

```diff
@@ -43,7 +43,7 @@ elif [ "$1" = "submodules-only" ]; then
 elif [ "$1" = "fast" ]; then
     git clone https://github.com/firesim/firesim-riscv-tools-prebuilt.git
     cd firesim-riscv-tools-prebuilt
-    git checkout 4a2f79d5c5a8a93f3e6a83bd32a0926cdb8983c5
+    git checkout 499867f3f2da4648ddaecf4dd7b1a59554c9d477
```

Comment:
I don't see this commit in riscv-tools-prebuilt. It looks like the correct hash (going by the commit comments) is:
db68e61662d1ad1ba622165f7be50e5606f5ec58

is there a more recent update that hasn't been pushed?

Reply:
Also, db68e6 does not include qemu while 4a2f79d does.

Reply:
The db68e6 has should be the correct one. I'm not sure why QEMU wasn't built, i'm trying a fresh full build now to see why.

Reply:
Yeah, so it seems the install-qemu script is responsible for building QEMU, it never happens in riscv-tools build.sh. I somehow broke install-qemu.sh when i manually compiled and installed a bunch of updated packages, not provided by rpm, to build openOCD. 

I think the right call is to provide a second build.sh script in firesim that simply skips building openOCD. That way we don't need a fork of riscv-tools, and we won't need to upgrade to package versions not provided by rpm.

Once i fix that, i'll add the ./install-qemu.sh script back to build-setup.sh. Alternatively, i could remove the additional qemu submodule and add the qemu installation to our firesim build.sh script.

Reply:
SGTM. I would advocate for adding it to the build-setup.sh script. The model I've been going on for firesim-software is that it needs some version of riscv-tools and qemu, but they aren't submodules. With this stuff all getting upstreamed, there is theoretically less of a need to submodule specific commits etc.

This isn't actually documented though, I should probably add that to the README.

Reply:
Updated.